### PR TITLE
Add consul role.

### PR DESCRIPTION
**Pull request by UmanShahzad**

### CODE REVIEW

File: deploy/requirements.yml [hunk 1]
```diff
@@ -39,6 +39,10 @@
   src: https://github.com/open-craft/ansible-forward-server-mail
   version: origin/master
 
+- name: consul
+  src: https://github.com/open-craft/ansible-role-consul
+  version: uman/consul
```
Comment (smarnach) [inline deploy/requirements.yml]: Just as a reminder, this needs to be change to the master branch once the ansible-role-consul PR has been merged.

Comment (UmanShahzad) [reply deploy/requirements.yml]: Yes, definitely.

**Comment by UmanShahzad** [comment]
@smarnach Since this is just reliant on the role itself, good to merge?

**Comment by UmanShahzad** [comment]
Builds failing randomly: rebuilding.

**Pull request merged**